daemon/start.go

```diff
@@ -45,7 +47,20 @@ func (daemon *Daemon) ContainerStart(name string, hostConfig *runconfig.HostConf
 	}
 
 	if err := container.Start(); err != nil {
-		return derr.ErrorCodeCantStart.WithArgs(name, utils.GetErrorMessage(err))
+		derrErr := derr.ErrorCodeCantStart.WithArgs(name, utils.GetErrorMessage(err))
```

Comment:
np, I was building on what was there already but I agree it should be in container.Start()